### PR TITLE
allow \n in pre

### PR DESCRIPTION
**Pull request by forresto**

### CODE REVIEW

**Comment by forresto** [comment]
@bergie pointers where this would be stripped? I only see `replace '&nbsp;', ' '` https://github.com/the-grid/html-flatten/blob/7857e957570efb5b602f3b4e06022803764cac2f/index.coffee#L477

**Comment by forresto** [comment]
Might be upstream issue: https://github.com/fb55/htmlparser2/issues/194 ... how to work around???

**Comment by bergie** [comment]
Then it might make sense to try to fix upstream

**Comment by forresto** [comment]
@bergie same test passes upstream: https://github.com/fb55/htmlparser2/issues/194#issuecomment-242419875

**Comment by bergie** [comment]
Then it is likely to be in our `tagToHtml` helper. Investigating

**Comment by bergie** [comment]
Hmm, no: even before `tagToHtml` the newline is gone:

``` javascript
[ { data: 'one two',                                                                                                                                                                                    
type: 'text',                                                                                                                                                                                       
next: null,                                                                                                                                                                                         
prev: null,                                                                                                                                                                                         
parent:                                                                                                                                                                                              
{ type: 'tag',                                                                                                                                                                                        
name: 'code',                                                                                                                                                                                       
attribs: {},                                                                                                                                                                                        
children: [Circular],                                                                                                                                                                               
next: null,                                                                                                                                                                                         
prev: null,                                                                                                                                                                                         
parent: [Object] } } ] 
```

**Comment by forresto** [comment]
@bergie I think I got it ☝️ .

**Comment by bergie** [comment]
@forresto cool! Any side-effects to be concerned with?

**Comment by forresto** [comment]
@bergie Tests are green, I think it's good to go. We were doing our own handling of br and nbsp;.

**Comment by bergie** [comment]
OK, will make a release then

File: index.coffee [hunk 1]
```diff
@@ -338,6 +334,7 @@ module.exports = class Flatten
           text: @tagToText tag
         results.push normalized
       when 'pre'
+        console.log tag, tag.children[0].children[0]
```
Comment (bergie) [inline index.coffee]: Can you remove this log statement?

**Pull request merged**